test.sh

```diff
@@ -1,10 +1,14 @@
-#!/usr/bin/env bash
-set -e
-for file in proto/*
+rootdir="$(dirname "$0")"
```

Comment:
I think don't need to remove the shebang.

Reply:
Thanks. I removed them by mistake...